test/manual/amp-tabs.amp.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP #0</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="../../dist/amp.js"></script>
+  <script async custom-element="amp-youtube" src="../../dist/v0/amp-tabs-0.1.max.js"></script>
+</head>
+<body>
+  <h1>AMP #0</h1>
+  <amp-tabs>
+    <ul>
```

Comment:
The ideal and most semantic format for this would be exactly the same as that of the accordion
<amp-tabs>
  <section open>
    <header>Heading</header>
     <div>Content</div>
  </section>

  <section>
    <header>Heading</header>
     <div>Content</div>
  </section>

  <section>
    <header>Heading</header>
     <div>Content</div>
  </section>
</amp-tabs>


Here is a page that achieves that by just using floats. (floats heading left and floats content right and clearfixes the container)


Also i think this proposed format is very easily achievable using amp-selector and amp bind? (or just amp-selector and css if we dont have to mutate height changes:( ). 

If you think about this this is just 2 carousels (one slide and other normal) - if not for semantics we probably might not end up needing this component